core/src/main/java/com/datastax/oss/driver/internal/core/channel/InFlightHandler.java

```diff
@@ -91,6 +95,21 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
     this.logPrefix = ownerLogPrefix + "|" + channelId.substring(1, channelId.length() - 1);
   }
 
+  @Override
+  public void channelWritabilityChanged(ChannelHandlerContext ctx) throws Exception {
+    System.out.println("Channel writability changed " + ctx.channel().isWritable());
+    if (ctx.channel().isWritable()) {
+      driverChannel.thenAccept(
+          d -> {
+            WriteCoalescer writeCoalescer = d.getWriteCoalescer();
+            if (writeCoalescer instanceof DefaultWriteCoalescer) {
+              ((DefaultWriteCoalescer) writeCoalescer).restartFlushers();
+            }
```

Comment:
It seems that this TCP needs to be implemented only in the `DefaultWriteCoalescer`.
In such a case, we don't need to change the interface of the flusher